microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -192,46 +173,47 @@ A typical JavaScript object can be one of two array types:
 
 When there are a small number of properties, the properties are stored internally in the JavaScript object.
 
-**Map** is an object that describes both the kind of object it is and the layout.  For example, maps are used to describe implicit object hierarchies for [fast property access](https://v8.dev/blog/fast-properties).
-
-### Object groups
+**Map** is an object that describes both the kind of object it is and the layout. For example, maps are used to describe implicit object hierarchies for [fast property access](https://v8.dev/blog/fast-properties).
```

Comment:
In the DevTools, this is called `system / Map`. `Map` is used to refer to JavaScript [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) objects.